vuepress/docs/tutorials/devops/default-database.md

```diff
@@ -0,0 +1,35 @@
+# Tutorial: Selecting the default databases
+
+## Overview
+
+This document provides a guide to changing an Entando environment by selecting the default databases used. 
+See [External Database](./external-database/) to connect to a database outside of the cluster.
+
+## Prerequisites
+
+-   An environment to install your Entando applicaiton
+-   Network access from your Kubernetes cluster to your database
+
+## Tutorial
+
+If needed you can set the desired component to use a specific DBMS by updating the Helm-generated yaml file (e.g. 'entando.yaml' from [Getting Started](/docs/getting-started]) as follows:
+
+1. open the chosen file and search for `EntandoCompositeApp` custom resource
+2. identify the component to update in the related list
+3. update the `spec.dbms` property with the desired value (you can specify different DBMS for different components)
+
+Repeat previous steps for all components you need to change used DBMS on.
+
+Valid values for `spec.dbms` property are: `none`, `postgresql`, `mysql`, `oracle`.
+
+`none` value will result in using an embedded database with in-file persistence strategy.
+If `none` (embedded database) is choosen each interested container requires a [PVC](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) mounted on path `/opt/jboss/keycloak/standalone/data/`
```

Comment:
```suggestion
```
If you agree, I would remove this info as really a detail. You can't customize the PVC anyway from a user point of view.